xml/rmt_install.xml

```diff
@@ -31,7 +33,232 @@
    possible for a single server to host both simultaneously.
   </para>
  </warning>
+ <sect1 xml:id="sec-rmt-storage-requirements">
+     <title>Storage Requirements</title>
+     <para>
+         Downloaded packages are stored in <filename>/var/lib/rmt/public/repo/SUSE/Products/</filename>. 
+         The amount of storage your &rmt; server requires is dependent on several variables: the 
+         number of  repositories and architectures that you mirror, how many packages are 
+         downloaded, the  number of  extensions that are enabled, and the age of your server installation. 
+         Outdated  packages are not removed by &rmt;, but accumulate over time. When new versions are 
+         downloaded, the old versions are kept on disk but are not referenced by repository 
+         metadata, which makes them invisible to clients.
+     </para>
+     <para>
+         As a general guide, two to three times the total size of all mirrored repositories should be 
+         sufficient. If you are mirroring a single release, such as &slea; 15.1, 250 GB provides room
+         for packages and updates. The total size of all x86_64 repositories for &slea; 12 SP4 is about 130 GB, and about   
+         155 GB for &slea; 15. There is not (yet) a tool for reporting repository sizes, so it is a good practice to monitor disk 
```

Comment:
Reporting repo sizes -- I'm uncertain if anything like this will be ever officially supported. So again, not sure if it is worth mentioning.